transport-native-kqueue/src/main/java/io/netty/channel/kqueue/Native.java

```diff
@@ -160,6 +164,24 @@ private static void loadNativeLibrary() {
         }
     }
 
+    private static boolean isSupportingFastOpenClient() {
+        try {
+            return fastOpenClient() == 1;
+        } catch (Exception e) {
+            logger.debug("Failed to probe fastOpenClient sysctl, assuming client-side TCP FastOpen cannot be used.", e);
+        }
+        return false;
+    }
+
+    private static boolean isSupportingFastOpenServer() {
+        try {
+            return fastOpenServer() == 1;
+        } catch (Exception e) {
+            logger.debug("Failed to probe fastOpenServer sysctl, assuming server-side TCP FastOpen cannot be used", e);
```

Comment:
missing dot at EOL (to make it consistent with the above log message.

```suggestion
            logger.debug("Failed to probe fastOpenServer sysctl, assuming server-side TCP FastOpen cannot be used.", e);
```